cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/goals/LeaderBytesInDistributionGoal.java

```diff
@@ -139,7 +139,7 @@ protected void rebalanceForBroker(Broker broker, ClusterModel clusterModel, Set<
 
     List<Replica> leaderReplicasSortedByBytesIn = broker.replicas().stream()
         .filter(r -> r.isLeader())
-        .filter(r -> !excludedTopics.contains(r.topicPartition().topic()))
+        .filter(r -> !shouldExclude(r, excludedTopics))
```

Comment:
This is worth discussing. The reason we are also excluding the leaders was because applications such as kafka-monitor needs to have full coverage of all the brokers. So we should not move leaders away from a broker for those topics.

So the meaning for excluded topics is the following:
1. When there is no dead brokers, those topics will not be touched by CC at all, whether for leader movement or replica movement.
2. When there is a dead broker, CC will only make necessary movements for the replicas of those excluded topics that are impacted by the broker failure.

In this particular case, if the original broker is dead, the replica/leader has to be moved anyways, so CC will not exclude those replicas.